docs/content/how-to/aws/disaster-recovery.md

```diff
@@ -1,17 +1,41 @@
 ---
-title: Migrating Hosted Cluster among the same AWS Region
+title: Disaster Recovery
 ---
 
-# Migrating Hosted Cluster among the same AWS Region
+# Disaster Recovery
 
-## Preface and Considerations
+## Migrating Hosted Cluster among the same AWS Region
+
+### Use cases for Disaster Recovery
+
+We want to help people to understand what situations could be considered a fit for this disaster recovery procedure.
+
+1. *The control-plane is down for your hosted cluster (api-server, etcd,...)*
```

Comment:
My preference would be that you list these under 2 headers:

This procedure is helpful for:
- description of case1
- description of case2

It is not helpful for:
- description of case3

Reply:
What do you think now? this looks better?